HdfsAccessor.go

```diff
@@ -118,13 +118,14 @@ func (this *hdfsAccessorImpl) connectToNameNodeImpl(nnAddr string) (*hdfs.Client
 
 // Opens HDFS file for reading
 func (this *hdfsAccessorImpl) OpenRead(path string) (ReadSeekCloser, error) {
```

Comment:
As discussed, "remove our own lock from MetadataClient" would be another workitem.